types/react-primitives/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
please do not add configurations to the rules. we would like to keep them all on the same rule set when possible.